### DISCUSSION TITLE
Need to suppress warning about not having memoized selectors.

### DISCUSSION THREAD

**Discussion by iamtylerd** [reduxjs/redux#4670] [Help] [answered]
I have 100's of warnings like this.
`Selector unknown returned a different result when called with the same parameters. This can lead to unnecessary rerenders.
Selectors that return a new reference (such as an object or an array) should be memoized`

We obviously plan / want to fix this, but it will be tech debt and something we can not do immediately.  Is there a way to suppress these warnings / or even better to not have redux run this check until we can get into a better state?

**Comment by aryaemami59** [comment, marked as answer]
You can disable it for all selectors globally:

```tsx
<Provider store={store} stabilityCheck="never">
    <App />
</Provider>
```

Or you can disable it per selector:

```tsx
const selectItems = useSelector((state: RootState) => state.items, {
  devModeChecks: { stabilityCheck: "never" },
})
```

You can read more about it [here](https://react-redux.js.org/api/hooks#selector-result-stability)

**Reply by iamtylerd** [reply]
Thanks a ton, spent a few hours reading documentation trying to find this information
